server/routes/v1/projects.py

```
@@ -0,0 +1,392 @@
+
+
+def compute_number_score(truth, prediction):
+    """
+    Simple numeric error-based score:
+    Score = 1 - (|truth - pred| / (|truth| + epsilon)).
+    """
+    try:
+        error = abs(float(truth) - float(prediction))
+        denom = abs(float(truth)) + 1e-8
+        return max(0, 1 - error / denom)
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)